plaidml/bridge/openvino/tests/functional/plaidml/single_layer_tests/activation_tests.cpp

```diff
@@ -1,328 +0,0 @@
-// Copyright (C) 2020 Intel Corporation
-// SPDX-License-Identifier: Apache-2.0
-//
-
-#include "gtest/gtest.h"
-
-#include "ie_core.hpp"
-#include "ie_plugin_ptr.hpp"
-#include "ir_gen_helper.hpp"
-
-#include "cpp/ie_cnn_net_reader.h"
```

Comment:
I'm removing these deprecated tests as this file no longer exists. It could probably be fixed, but since the tests are deprecated anyway, I'm thinking it might not be worth it. This does leave us with just one main path through the OV code, rather than 2, but I'm hoping that problem will be addressed soon by getting some full networks running.